### PR TITLE
Unlock with fingerprint on notification double tap without pin/passwa…

### PR DESCRIPTION
**Pull request by Hikari-no-Tenshi**
…rd/pattern [2/2]

### CODE REVIEW

**Review by gwolf2u — APPROVED** [review]

**Comment by gwolf2u** [comment]
good to go here

**Comment by gwolf2u** [comment]
@neobuddy89 need to change to something else or merge as is?

**Comment by neobuddy89** [comment]
@gwolf2u @Hikari-no-Tenshi Can we please make title string more crisp and summary string more elaborate? I am still unsure if this feature is useful for non-FOD devices... but let me test it more.

File: res/values/cr_strings.xml [hunk 1]
```diff
@@ -2524,4 +2524,8 @@
 
     <!-- Sound: Title for the option defining the phone ringtone for slot2 on Muilti SIM device. [CHAR LIMIT=50] -->
     <string name="ringtone2_title">Phone ringtone - SIM 2</string>
+
+    <!-- Fingerprint unlock without bouncer -->
+    <string name="unlock_without_bouncer_title">Fingerprint unlock on notification double tap</string>
```
Comment (gwolf2u) [inline res/values/cr_strings.xml]: propose string change (or something similar)
"Use fingerprint unlock to view notification"

File: res/values/cr_strings.xml [hunk 2]
```diff
@@ -2524,4 +2524,8 @@
...
+    <string name="unlock_without_bouncer_summary">Unlock device with fingerprint on notification double tap without pin/password/pattern</string>
```
Comment (gwolf2u) [inline res/values/cr_strings.xml]: and here add "without need to input"
"Unlocks device with fingerprint on notification double tap without need to input pin/password/pattern"

**Review by gwolf2u — CHANGES_REQUESTED** [review]

**Review by gwolf2u — APPROVED** [review]

File: res/xml/crdroid_settings_lockscreen.xml [hunk 3]
```diff
@@ -251,6 +251,13 @@
             android:summary="@string/fp_wake_and_unlock_summary"
             android:persistent="false" />-->
 
+
+        <com.crdroid.settings.preferences.SecureSettingSwitchPreference
```
Comment (neobuddy89) [inline res/xml/crdroid_settings_lockscreen.xml:255]: SecureSettingSwitchPreference

File: src/com/crdroid/settings/fragments/LockScreen.java [hunk 4]
```diff
@@ -131,6 +139,8 @@ public static void reset(Context mContext) {
                 Settings.System.FP_ERROR_VIBRATE, 1, UserHandle.USER_CURRENT);
         Settings.System.putIntForUser(resolver,
                 Settings.System.FP_SUCCESS_VIBRATE, 1, UserHandle.USER_CURRENT);
+        Settings.System.putIntForUser(resolver,
```
Comment (neobuddy89) [inline src/com/crdroid/settings/fragments/LockScreen.java:142]: Settings.System.putIntForUser

**Pull request merged**